rc/detection/file.kak

```diff
@@ -22,5 +22,5 @@ define-command -hidden file-detection %{ evaluate-commands %sh{
     fi
 } }
 
-hook global BufOpenFile .* file-detection
-hook global BufWritePost .* file-detection
+hook -group file-detection global BufOpenFile .* file-detection
+hook -group file-detection global BufWritePost .* file-detection
```

Comment:
Thanks for this patch. I sometimes edit plaintext files that are wrongly classified as C++ or Ruby by file(1). It'll be convenient to work around this via disabled_hooks.

`file-detection` is a good name IMO. Looking at our other hooks, alternatives are `file-hooks` or `file` but both sound pretty generic